README.md

```diff
@@ -75,4 +76,5 @@ You can either just use the similar command line parameters (see `signal-media-e
 
 * `outputDir`: Output directory for media files, as a relative or absolute path. Inside this directory, subdirectories for the sender numbers or names will be created.
 * `includeExpiringMessages`: Whether to include media files from messages that are set to expire in the future.
+* `includeAttachments`: Whether to include "all" attachments (including audio) or just "visual" attachments.
```

Comment:
I did not test it, but I suspect "all" might also include giphy gifs sent from the Android app. Just something that could be mentioned here.

Reply:
Both "all" and "visual" include GIFs. If it didn't I would consider that a bug. GIFs are visual.